torch/_torch_docs.py

```diff
@@ -5810,6 +5810,51 @@ def merge_dicts(*dicts):
     tensor([  435.,  1335.,  2235.,  3135.])
 """.format(**multi_dim_common))
 
+add_docstr(torch.nansum,
+           r"""
+nansum(input, dtype=None) -> Tensor
+
+Return the sum of array elements over a given axis treating Not a Numbers (NaNs) as zero.
+
+Args:
+    {input}
+    {dtype}
+
+Example::
+
+    >>> a = torch.randn(1, 3)
+    >>> a
+    tensor([[ 0.1133, -0.9567,  0.2958]])
+    >>> torch.nansum(a)
+    tensor(-0.5475)
+
+.. function:: nansum(input, dim, keepdim=False, dtype=None) -> Tensor
+
+Returns the sum of each row of the :attr:`input` tensor in the given
+dimension :attr:`dim` treating Not a Numbers (NaNs) as zero. 
+If :attr:`dim` is a list of dimensions, reduce over all of them.
+
+{keepdim_details}
+
+Args:
+    {input}
+    {dim}
+    {keepdim}
+    {dtype}
```

Comment:
dtype is an optional kwarg that can't be specified positionally

Reply:
Updated. 

Note: `torch.sum` also has the same structure. Should be updated.